### PR TITLE
Add poolboy usage info to README

### PR DESCRIPTION
**Pull request by JGulbronson**
As per https://github.com/kafkaex/kafka_ex/issues/137

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,22 @@ iex> KafkaEx.create_worker(:no_name) # indicates to the server process not to na
 {:ok, #PID<0.171.0>}
 ```
 
+### Using KafkaEx with a pooling library
+
+Note that KafkaEx has a supervisor to manage it's workers. If you are using Poolboy or a similar
```
Comment (dantswain) [inline README.md]: "its"

**Comment by dantswain** [comment]
@JGulbronson Do you feel that the original wording you cited in the README is still confusing? If so please feel free to modify that or replace it with a "see below" or whatever.

**Comment by JGulbronson** [comment]
I think it's fine, since I made sure to keep the sections side-by-side, and creating an unnamed worker isn't just applicable to Poolboy. At the very least, this should allow people to not get stuck where I did.

**Comment by dantswain** [comment]
Looks good to me!  @bjhaid do you have time to merge?  I'm travelling this week, so it's iffy.

**Pull request merged**